### PR TITLE
Fix orphan wells

### PR DESCRIPTION
**Pull request by misterhay**
Fix a typo and an SSL error

### CODE REVIEW

File: SocialStudies/OrphanWells/orphan-wells.ipynb [hunk 1]
```diff
@@ -112,7 +112,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "url = 'https://www.wellwiki.org/wiki/Special:AlbertaAdvancedWellSearch?operatorfiltertype=matches&operatorfiltervalue=&countyfiltertype=matches&countyfiltervalue=&datefiltertype=matches&datefiltervalue=&licensefiltertype=matches&licensefiltervalue=Abandoned&meridianfiltertype=matches&meridianfiltervalue=&townshipfiltertype=matches&townshipfiltervalue=&rangefiltertype=matches&rangefiltervalue=&sectionfiltertype=matches&sectionfiltervalue=&pagenum=&pagesize=100000'\n",
+    "url = 'http://www.wellwiki.org/wiki/Special:AlbertaAdvancedWellSearch?operatorfiltertype=matches&operatorfiltervalue=&countyfiltertype=matches&countyfiltervalue=&datefiltertype=matches&datefiltervalue=&licensefiltertype=matches&licensefiltervalue=Abandoned&meridianfiltertype=matches&meridianfiltervalue=&townshipfiltertype=matches&townshipfiltervalue=&rangefiltertype=matches&rangefiltervalue=&sectionfiltertype=matches&sectionfiltervalue=&pagenum=&pagesize=100000'\n",
```
Comment (meghbali) [inline SocialStudies/OrphanWells/orphan-wells.ipynb:115]: Please check the url. I get errors running the code cell both on Callysto hub and jupyter notebook.

**Review by meghbali — CHANGES_REQUESTED** [review]
Everything looks good, except a url that cannot be read (see the comment)

File: SocialStudies/OrphanWells/orphan-wells.ipynb (hunk 1 above)
Comment (ZachAttacksProblems) [reply SocialStudies/OrphanWells/orphan-wells.ipynb:115]: The URL works, but there's something about the SSL signing that is wrong. I didn't have any luck with the http (no s), but you can get it to work with the requests library first which allows you to ask it to ignore the SSL error. 

Working on a fix, can add the change this afternoon.

Comment (ZachAttacksProblems) [reply SocialStudies/OrphanWells/orphan-wells.ipynb:115]: @meghbali Can you verify that altering the cell as follows corrects the issue? 

```
url = 'https://www.wellwiki.org/wiki/Special:AlbertaAdvancedWellSearch?operatorfiltertype=matches&operatorfiltervalue=&countyfiltertype=matches&countyfiltervalue=&datefiltertype=matches&datefiltervalue=&licensefiltertype=matches&licensefiltervalue=Abandoned&meridianfiltertype=matches&meridianfiltervalue=&townshipfiltertype=matches&townshipfiltervalue=&rangefiltertype=matches&rangefiltervalue=&sectionfiltertype=matches&sectionfiltervalue=&pagenum=&pagesize=100000'

#import ssl
#ssl._create_default_https_context = ssl._create_unverified_context

# import pandas as pd
# ww = pd.read_html(url)[1]
# ww.columns = ['UWI','Operator','County/Municipality','Meridian','Township','Range','Section','License Date','License Status']
# ww

import requests, warnings
import pandas as pd

with warnings.catch_warnings():
    warnings.simplefilter("ignore")
    res = requests.get(url, verify=False)
ww = pd.read_html(res.text)[1]
ww.columns = ['UWI','Operator','County/Municipality','Meridian','Township','Range','Section','License Date','License Status']
ww
```

Comment (meghbali) [reply SocialStudies/OrphanWells/orphan-wells.ipynb]: @ZachAttacksProblems: Thanks. I implemented the changes you suggested and the issue is now resolved. I pushed a new commit.

**Review by meghbali — APPROVED** [review]

File: SocialStudies/OrphanWells/orphan-wells.ipynb [hunk 2]
```diff
@@ -112,10 +112,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import warnings, requests\n",
     "url = 'https://www.wellwiki.org/wiki/Special:AlbertaAdvancedWellSearch?operatorfiltertype=matches&operatorfiltervalue=&countyfiltertype=matches&countyfiltervalue=&datefiltertype=matches&datefiltervalue=&licensefiltertype=matches&licensefiltervalue=Abandoned&meridianfiltertype=matches&meridianfiltervalue=&townshipfiltertype=matches&townshipfiltervalue=&rangefiltertype=matches&rangefiltervalue=&sectionfiltertype=matches&sectionfiltervalue=&pagenum=&pagesize=100000'\n",
     "\n",
-    "import pandas as pd\n",
-    "ww = pd.read_html(url)[1]\n",
+    "#import ssl\n",
```
Comment (ZachAttacksProblems) [inline SocialStudies/OrphanWells/orphan-wells.ipynb:118]: Should probably delete these commented/unused lines

Comment (meghbali) [reply SocialStudies/OrphanWells/orphan-wells.ipynb:118]: I thought there were put there intentionally for having different options. I will remove them.

Comment (ZachAttacksProblems) [reply SocialStudies/OrphanWells/orphan-wells.ipynb]: I don't think so... I think we're good to just get rid of them.

**Review by ZachAttacksProblems — APPROVED** [review]

**Comment by misterhay** [comment]
This is great, thank you for your work on it.

**Pull request merged**